src/main/java/services/gorest/stepdefinition/UserStepsDefinition.java

```diff
@@ -0,0 +1,101 @@
+package services.gorest.stepdefinition;
+
+import cucumber.api.Scenario;
+import cucumber.api.java.After;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import io.restassured.response.Response;
+import net.thucydides.core.annotations.Steps;
+import net.thucydides.core.annotations.WithTag;
+import net.thucydides.core.annotations.WithTags;
+import services.gorest.actions.user.CreateUser;
+import services.gorest.actions.user.DeleteUser;
+import services.gorest.actions.user.GetUser;
+import services.gorest.actions.user.UpdateUser;
+import services.gorest.models.responses.GetUserResponse;
+import services.gorest.validation.CommonValidations;
+import utils.methods.ReusableMethods;
+
+import static utils.variables.SessionVariableManager.getSessionVariable;
+import static utils.variables.SessionVariableManager.setSessionVariable;
+import static utils.variables.SessionVariables.*;
+
+@WithTags({
+        @WithTag(type = "service", name = "GoRest"),
+        @WithTag(type = "type", name = "Smoke"),
+        @WithTag(type = "type", name = "Regression")
+})
+public class UserStepsDefinition {
+
+    @Steps
+    private CreateUser createUser;
+
+    @Steps
+    private GetUser getUser;
+
+    @Steps
+    private DeleteUser deleteUser;
+
+    @Steps
+    private UpdateUser updateUser;
+
+    @Steps
+    private CommonValidations commonValidations;
+
+    @Steps
+    private ReusableMethods reusableMethods;
+
+
+    @When("^I create a new user with email (.*), first name (.*), last name (.*) and gender (.*)$")
+    public void whenCreateUser(String email, String firstName, String lastName, String gender) {
+        Response response = createUser.whenCreateNewUser(email, firstName, lastName, gender);
+        setSessionVariable(VAR_USER, response.as(GetUserResponse.class).getResult());
```

Comment:
hint: I would create a new object representing the GetUserResponse instead of deserializing it everytime.
`GetUserResponse getUserResponse = response.as(GetUserResponse.class);
setSessionVariable(VAR_USER, getUserResponse.getResult());`

This way is easy to read too